fix: minor information to docs (#290)
aerogear/offix: docs/angular.md

---
id: angular
title: Angular
sidebar_label: Angular
---

> Offix is not providing speciallized API for Angular platform,

Reviewer: ```suggestion
Offix does not provide a specific API for Angular,
```

> however deverlopers can use our vanilla js client for the most of the tasks.

Reviewer: ```suggestion
However, the regular `offix-client` module can be used for most use cases.
```

## Example Angular Application

> Offix is implemented in Angular based Ionic example application.

Reviewer: ```suggestion
We maintain an Angular based example Ionic application that showcases how to use Offix.
```
